<ui version="4.0" >
 <class>WindowDefinitionListWidget</class>
 <widget class="QWidget" name="WindowDefinitionListWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QListWidget" name="list" />
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" >
     <item>
      <widget class="QPushButton" name="edit_button" >
       <property name="text" >
        <string>&amp;Edit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="new_button" >
       <property name="text" >
        <string>&amp;New</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="duplicate_button" >
       <property name="text" >
        <string>&amp;Duplicate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delete_button" >
       <property name="text" >
        <string>&amp;Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
